app/controllers/result_filters/location_controller.rb

```diff
@@ -2,21 +2,89 @@ module ResultFilters
   class LocationController < ApplicationController
     include FilterParameters
 
+    NO_OPTION = nil
+    LOCATION_OPTION = "1".freeze
```

Comment:
available options 

https://github.com/DFE-Digital/search-and-compare-ui/blob/37c4385ea5764a05743078d23012122524e45423/src/Filters/Enums/LocationOption.cs

Reply:
This is about the value of the `l` parameter. If it's nil then no option was selected. If 1, `Across England` was selected ...

There might be a need to revisit this once @faissalMT 's work is merged